packages/pool/contracts/StateUtils.sol

```diff
@@ -36,6 +36,9 @@ contract StateUtils is IStateUtils {
     string constant internal ERROR_VALUE = "Invalid value";
     string constant internal ERROR_ADDRESS = "Invalid address";
     string constant internal ERROR_UNAUTHORIZED = "Unauthorized";
+    string constant internal ERROR_FREQUENCY = "Try again a week later";
```

Comment:
Since this is a weird error, I figured it would be good to have a more helpful error string